Depend/bump

JLambertazzo opened the pull request:
# Description
Updated dependencies.

* Updated `@rollup/plugin-commonjs` major version to [V22](https://github.com/rollup/plugins/releases/tag/commonjs-v22.0.2)
* Not updated `jest` or `ts-jest` to new major version V29 since `ts-jest` latest version is still depending on `jest` V28

### Test Coverage
- [x] unit

#### References
- Closes ```GRI-90```

bernachagas reviewed (changes requested):
I think you may still need to run `npm update` with the `--save` flag. The `package-lock.json` file was correctly updated, but the packages `package.json` should have been updated as well.

bernachagas commented:
Since this is one of our open source repos, it's always good to confirm in the description that, besides the unit tests, you've also done the regression testing (see item 4 on [this document](https://docs.google.com/document/d/1CrxCbQbcEF6wAA4q2r09Jxb-SaNtgCAqupRRXntLbvA/edit)).

JLambertazzo commented:
Updated again with `--save` flag, and bumped some dependencies that have updated since the PR was made. Also ran check-licenses and found no change in output. Updated PR description with all of this 👍
